kumascript/macros/XULElem.ejs

```diff
@@ -11,12 +11,6 @@ var destOldType = "/" + lang + '/docs/XUL:' + $0;
 var destNewType = "/" + lang + '/docs/Mozilla/Tech/XUL/' + $0;
 var url = "";
 
-if (wiki.pageExists(destNewType)) {
-    url = destNewType;
-} else if (wiki.pageExists(destOldType)) {
-    url = destOldType;
-} else {
-    url = destNewType;
-}
+url = destNewType;
```

Comment:
At this point, combine with line 11 and 12 and just 
```suggestion
var url = "/" + lang + '/docs/Mozilla/Tech/XUL/' + $0;
```

Reply:
Will do!